pulsar-functions/instance/src/main/python/python_instance_main.py

```diff
@@ -74,6 +73,11 @@ def main():
   args = parser.parse_args()
   function_details = Function_pb2.FunctionDetails()
   json_format.Parse(args.function_details, function_details)
+
+  if os.path.splitext(str(args.py))[1] == '.whl':
+    os.system("unzip -d %s -o %s" % (os.path.dirname(str(args.py)), str(args.py)))
```

Comment:
unzip is a new dependency.

Reply:
where is this unzipped to?

Reply:
it is unzipped into the same tmp directory the .whl file is copied to. The -d option controls that.

Reply:
unzip is indeed a new dependency.

Reply:
`unzip` might not be always installed (especially in Docker images where everything is stripped to bone). 
Python has a native way to deal with zip files, without invoking the CLI command: https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.extract

Reply:
I have removed the unzip dependency by switching over to zipfile.